vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/dataview/ComboBoxListDataView.java

```diff
@@ -119,4 +120,66 @@ public Registration addItemCountChangeListener(
             ComponentEventListener<ItemCountChangeEvent<?>> listener) {
         return super.addItemCountChangeListener(listener);
     }
+
+    /**
+     * Adds a filter to be applied to all queries. The filter will be used in
+     * addition to any filter that has been set or added previously. The filter
+     * is applied on data view and is different from the filter string that can
+     * be typed in by the user in UI.
+     * <p>
+     * A filter bound to data set, not to the component. That means this filter
```

Comment:
I propose to change this paragraph right now as it done here https://github.com/vaadin/flow/pull/9315/commits/600581253bf56ec9740076c139f3372b2fd1fe5b# because otherwise we will need create another PR later on.

Reply:
Done.

Reply:
Let's modify it a bit more, according to that fix https://github.com/vaadin/flow/pull/9315/commits/3fac591ef499bfd7e2c71db0206e764873a71c7b

The problem is that these changes are in different classes, so we can overlook them in the future when both PR are merged.

Reply:
No problem. Changed that part of javadoc according to [vaadin/flow@3fac591](https://github.com/vaadin/flow/commit/3fac591ef499bfd7e2c71db0206e764873a71c7b) for `setFilter` and `addFilter` methods.
Done.